pkg/util/permissions/permissions.go

```diff
@@ -11,32 +11,37 @@ import (
 func CanDoAction(ps []authorization.Permission, a string) (bool, error) {
 	for _, p := range ps {
 		var matched bool
-		for _, action := range *p.Actions {
-			action := regexp.QuoteMeta(action)
-			action = "(?i)^" + strings.ReplaceAll(action, `\*`, ".*") + "$"
-			rx, err := regexp.Compile(action)
-			if err != nil {
-				return false, err
-			}
-			if rx.MatchString(a) {
-				matched = true
-				break
+
+		if p.Actions != nil {
```

Comment:
not a fan of this deep indenting

Reply:
I gave @m1kola a bad steer on this - my bad.  I have reverted my suggestion and pushed the fix.